### ISSUE TITLE
POST on /runs fails

### ISSUE THREAD

**Issue by ericwb** [vmware/column#59]
POST http://10.20.140.104:48620/runs
Body:
```
{
	"playbook_path": "/home/ericwb/column/tests/fixtures/playbooks/hello_world.yml"
}
```

```
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/local/lib/python2.7/dist-packages/column-0.3.6-py2.7.egg/column/api/manager/run_manager.py", line 77, in _run_playbook
    **column_opts['options'])
KeyError: 'options'
```

**Comment by ushergod** [comment]
https://github.com/vmware/column/pull/60

**Comment by ericwb** [comment]
Fixed in commit #60
